OpenSim/Sandbox/MatlabScripts/osimTableToStruct.m

```diff
@@ -0,0 +1,91 @@
+function data = osimTableToStruct(table)
+%% // Convert Matlab Struct to OpenSim time Series Table
+%  Input is an OpenSim TimesSeriesTable
+%
+%  Output is a Maltab stucture where data.label = n X 1 or n x 3 array
+%  ie structdata.LASI = [2 3 4; 4 5 6, ...
+
+% ----------------------------------------------------------------------- %
+% The OpenSim API is a toolkit for musculoskeletal modeling and           %
+% simulation. See http://opensim.stanford.edu and the NOTICE file         %
+% for more information. OpenSim is developed at Stanford University       %
+% and supported by the US National Institutes of Health (U54 GM072970,    %
+% R24 HD065690) and by DARPA through the Warrior Web program.             %
+%                                                                         %
+% Copyright (c) 2005-2012 Stanford University and the Authors             %
+% Author(s): James Dunne                                                  %
+%                                                                         %
+% Licensed under the Apache License, Version 2.0 (the "License");         %
+% you may not use this file except in compliance with the License.        %
+% You may obtain a copy of the License at                                 %
+% http://www.apache.org/licenses/LICENSE-2.0.                             %
+%                                                                         %
+% Unless required by applicable law or agreed to in writing, software     %
+% distributed under the License is distributed on an "AS IS" BASIS,       %
+% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or         %
+% implied. See the License for the specific language governing            %
+% permissions and limitations under the License.                          %
+% ----------------------------------------------------------------------- %
+
+% Author: James Dunne, Shrinidhi K. Lakshmikanth, Chris Dembia, Tom Uchida,
+% Ajay Seth, Ayman Habib, Jen Hicks.
+
+%%
+import org.opensim.modeling.*
+
+%%
+nCol = table.getNumColumns;
+nRow = table.getNumRows;
```

Comment:
Prefer `getNumColumns()` and `getNumRows()` (with trailing parens) so it's clear these are method calls.